### ISSUE TITLE
Chore - all - Remove utility class references from examples

### ISSUE THREAD

**Issue by srambach** [patternfly/patternfly#5195]
**Describe the problem**
Some component examples are using utility classes. This has forced the React components to pull in utilities they shouldn't have to.

`.pf-u-screen-reader` is used in the following two instances, but should be using the global class `.pf-screen-reader`
https://github.com/patternfly/patternfly-react/blob/4c1b8219539ce61aa45bfefc935af732129404a5/packages/react-core/src/components/Popover/PopoverHeader.tsx#L42

https://github.com/patternfly/patternfly-react/blob/4c1b8219539ce61aa45bfefc935af732129404a5/packages/react-core/src/components/Modal/ModalBoxTitle.tsx#L67

Besides fixing these two instances, we should also replace any other instances of component examples using utility classes.

**Comment by srambach** [comment]
Probably still valid.
